src/js/utils/captions.js

```diff
@@ -253,64 +253,38 @@ export const isCCOn = (el) => {
-    if (subtitlesShowingStr && !el.hasAttribute('no-subtitles-fallback')) {
+    // If we have currently showing subtitles track(s), request for them to be disabled.
+    if (subtitlesShowingStr) {
       const evt = new window.CustomEvent(
         MediaUIEvents.MEDIA_DISABLE_SUBTITLES_REQUEST,
         { composed: true, bubbles: true, detail: subtitlesShowingStr }
       );
       el.dispatchEvent(evt);
     }
   } else if (!forceOff) {
-    // Closed Captions is off. Clicking should show the first relevant captions track or subtitles track if we're using subtitle fallback (true/"on" by default)
-    const [ccTrackStr] =
+    // Subtitles are off. Clicking should show the first relevant captions track or subtitles track (true/"on" by default)
+    const [subTrackStr] =
       splitTextTracksStr(
-        el.getAttribute(MediaUIAttributes.MEDIA_CAPTIONS_LIST) ?? ''
+        el.getAttribute(MediaUIAttributes.MEDIA_SUBTITLES_LIST) ?? ''
       ) ?? [];
-    if (ccTrackStr) {
-      // If we have at least one captions track, request for the first one to be showing.
+    if (subTrackStr) {
+      // If we have at least one subtitles track (and didn't have any captions tracks), request for the first one to be showing as a fallback for captions.
       const evt = new window.CustomEvent(
-        MediaUIEvents.MEDIA_SHOW_CAPTIONS_REQUEST,
-        { composed: true, bubbles: true, detail: ccTrackStr }
+        MediaUIEvents.MEDIA_SHOW_SUBTITLES_REQUEST,
+        { composed: true, bubbles: true, detail: subTrackStr }
       );
       el.dispatchEvent(evt);
-    } else if (!el.hasAttribute('no-subtitles-fallback')) {
-      // If we don't have a captions track and we're using subtitles fallback (true/"on" by default), check if we have any subtitles available.
-      const [subTrackStr] =
-        splitTextTracksStr(
-          el.getAttribute(MediaUIAttributes.MEDIA_SUBTITLES_LIST) ?? ''
-        ) ?? [];
-      if (subTrackStr) {
-        // If we have at least one subtitles track (and didn't have any captions tracks), request for the first one to be showing as a fallback for captions.
-        const evt = new window.CustomEvent(
-          MediaUIEvents.MEDIA_SHOW_SUBTITLES_REQUEST,
-          { composed: true, bubbles: true, detail: subTrackStr }
-        );
-        el.dispatchEvent(evt);
-      }
-    } else {
-      // If we end up here, it means we have an enabled CC-button that a user has clicked on but there are no captions and no subtitles (or we've disabled subtitles fallback).
-      console.error(
-        'Attempting to enable closed captions but none are available! Please verify your media content if this is unexpected.'
-      );
     }
+  } else {
+    // If we end up here, it means we have an enabled CC-button that a user has clicked on but there are no captions and no subtitles (or we've disabled subtitles fallback).
```

Comment:
nitpick: Comment language is no longer accurate. Consider:
```suggestion
    // If we end up here, it means we have an enabled CC-button that a user has clicked on but there are no captions and no subtitles.
```

Reply:
Updated this as well as the error.